EnglishFilter/sections/antiadblock.txt

```diff
@@ -623,6 +623,8 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/95200
+calculator-online.net#%#//scriptlet('prevent-setTimeout', 'adsbygoogle')
```

Comment:
better:
```
||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,domain=calculator-online.net,important
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=calculator-online.net
```

Reply:
On Firefox with AdGuard Browser Extension, These don't work well.

Reply:
@piquark6046 
I don't see the adsbygoogle request at all in Firefox and I don't see antiadblock either. What do you see in Firefox?